### PR TITLE
Issue template redesign

### PR DESCRIPTION
**Pull request by 927589452**
Added a few more suggestions

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_report.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ labels: bug
...
-**Describe the issue**
-A clear and concise description of the issue.
+**Issue**
+<!--A clear and concise description of the issue in a few sentences. Try to stick to simple english, to make easier for others to understand.-->
 
-**Steps to reproduce**
+<!-- Provide the following files and commands in detail, so everybody can reproduce the issue-->
 
-*minimalvimrc*
+**minimalvimrc**
```
Comment (lervag) [inline .github/ISSUE_TEMPLATE/issue_report.md]: Is it better to call this `test.vim`, or perhaps with space, `minimal vimrc`? Or `minimal.vim`?

File: .github/ISSUE_TEMPLATE/issue_report.md [hunk 2]
```diff
@@ -46,13 +46,13 @@ A clear and concise description of the issue.
 - If you are a neovim user, start neovim with `nvim -u minivimrc minimal.tex`
 -->
 
-*Commands/Inputs*
+**Commands/Inputs**
```
Comment (lervag) [inline .github/ISSUE_TEMPLATE/issue_report.md]: I'd remove the `s` from `Inputs`.

File: .github/ISSUE_TEMPLATE/issue_report.md (hunk 1 above)
Comment (927589452) [reply .github/ISSUE_TEMPLATE/issue_report.md]: I like `minimal.vim` and will provide a commit for that (also fixing the commands)

**Comment by lervag** [comment]
Thanks! I've merged this now.

**Pull request closed without merging**